lib/datasource/api.ts

```diff
@@ -6,6 +6,7 @@ import { CdnJsDatasource } from './cdnjs';
 import { ClojureDatasource } from './clojure';
```

Comment:
If you want, you can add yourself to the list of `contributors` in the `package.json` file. 😉 

https://github.com/renovatebot/renovate/blob/cc016ac14c3aa3183825b16e67a40b983def9adf/package.json#L78-L116